bimmer_connected/vehicle/vehicle.py

```diff
@@ -300,25 +300,25 @@ async def get_vehicle_image(self, direction: VehicleViewDirection) -> bytes:
     # Deprecated
     # # # # # # # # # # # # # # #
 
-    @property
+    @property  # type: ignore
```

Comment:
I would suggest specifying specific error codes with `type: ignore`. In this case, I believe it should be `# type: ignore[misc]`.

There are two mypy settings which can be helpful. We've enabled both for HA.
```ini
[mypy]
show_error_codes = true
enable_error_code = ignore-without-code
```